docs/applications/pyserver/index.rst

```diff
@@ -0,0 +1,16 @@
+Proxy-Server (python)
+=======================================
+
+- a Python server that runs on the Receivers machine
+
+
+
+.. toctree::
+   :maxdepth: 2 
+
+   installation
+   usage
+   cmdline
+   
```

Comment:
I see there is no `py-api.rst` - why not?

Reply:
You're right. It's missing for some reason.
Fixed with: https://github.com/raiden-network/microraiden/commit/ba1c77f65ffd4f179116a551ed9b3f92a85ff6ce